new-frontiers-14/frontier-station-14
# HealingSystem: Check bleeding or restoring blood in HasDamage.

## About the PR



Allows healing items to repeatedly heal if they staunch bleeding or restore blood on a target that isn't damaged in a relevant way (e.g. using gauze or a blood pack on a bleeding body with 50% blood volume, but only radiation damage)

## Why / Balance


https://discord.com/channels/1123826877245694004/1307968742340427828

Checks were inconsistent - one of two existing ones looked at restoring blood, neither looked at staunching blood.  With different maxBleedAmounts, this becomes much more relevant.

## How to test


1. Spawn Urist.
2. Spawn bang stick, 200 dmg
3. Bonk Urist.
4. Check damage.
5. Apply Gauze.
6. Bleeding should stop, blunt damage should be untouched.
7. Apply blood pack.
8. Blood should be restored, you should be able to apply a blood pack multiple times until all blood is restored.

## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Blood restoring/hemostatic items can be used even if the user isn't damaged.
